grid_apps/block_model_to_octree/driver.py

```diff
@@ -33,13 +33,24 @@
 logger = logging.getLogger(__name__)
 
 
-class Driver(BaseGridDriver):
+class Driver(BaseDriver):
     """
     Convert a BlockModel object to Octree with various refinement strategies.
     """
 
     _params_class = BlockModel2OctreeOptions
 
+    def run(self):
+        """Create an octree mesh from input values."""
+        with fetch_active_workspace(self.params.geoh5, mode="r+"):
+            logger.info("Converting BlockModel to Octree mesh . . .")
+            octree = self.make_grid()
```

Comment:
Q1: when you call `octree.make_grid`, does it saves to ` outgroup` if present?
From this code, it's not obvious when you are calling `outgroup`or `octree`